.cirrus.yml

```diff
@@ -380,16 +377,18 @@ bindings_task:
         (changesInclude('**/*.go', '**/*.c', '**/*.h') && !changesIncludeOnly('test/**', 'pkg/machine/e2e/**'))
     depends_on: &build
         - build_success
-    gce_instance: *standardvm
+    gce_instance: &standardvm
+      <<: *fastvm
+      cpu: 2
     env:
         <<: *stdenvars
         TEST_FLAVOR: bindings
     # N/B: This script depends on ${DISTRO_NV} being defined for the task.
     clone_script: &get_gosrc |
         cd /tmp
-        echo "$ARTCURL/Build%20for%20${DISTRO_NV}/repo/repo.tbz"
-        time $ARTCURL/Build%20for%20${DISTRO_NV}/repo/repo.tbz
-        time tar xjf /tmp/repo.tbz -C $GOSRC
+        echo "$ARTCURL/Build%20for%20${DISTRO_NV}/repo/repo.tar.zst"
+        time $ARTCURL/Build%20for%20${DISTRO_NV}/repo/repo.tar.zst
+        time tar --zstd -xf /tmp/repo.tar.zst -C $GOSRC
```

Comment:
From `info tar`:
>Reading compressed archive is even simpler: you don't need to specify
>any additional options as GNU ‘tar’ recognizes its format automatically.

I have a slight predilection for eliminating the `--zstd`, and just letting `tar` figure it out itself. WDYT?

Reply:
I can try that and see if it works, the one thing that really messed me up is that we extract on macos as well and there is no gnu tar there so I am not sure if that can figure it out too. If not we end up with a case where one tar command has it and the other does not which might cause more confusion.

Reply:
Oh good point. I hate macs so much.